config/routes.rb

```diff
@@ -16,7 +16,9 @@
   put 'user_reports/undo_delete/:id' => 'user_reports#undo_delete', as: :undo_delete_users_report
 
   resources :pages, only: [:index, :new, :create, :edit]
-  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy]
+  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy] do
+    get 'search', on: :collection
```

Comment:
@tansaku for the organisation search currently we are using https://github.com/AgileVentures/LocalSupport/blob/develop/config/routes.rb#L6 , normally the route accepts only **GET** 
The advantage of using the highlighted syntax is to regroup routes by resources instead of adding routes separately.
I suggested that we can refactor some routes like instead of using:
`resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy]` we can use `resources :volunteer_ops, :except => [:new, :create]`